### DISCUSSION TITLE
How to call `trace()` in .hxc files?

### DISCUSSION THREAD

**Discussion by MAZ12211** [FunkinCrew/Funkin#3704] [Q&A] [answered]
Calling `trace()` in v0.5.1/2 doesn't work anymore as it used to
So far I've been using `PolymodErrorHandler.showAlert` but that won't work well at all if it's in `update` function if I want to check for collision or something
I tried running the game through Vscode terminal with & without `-debug` at the end
Anybody knows a solution?

**Comment by lemz1** [comment]
they made traces only work in debug mode

are you sure you compiled for debug mode?

**Reply by MAZ12211** [reply]
> they made traces only work in debug mode
> 
> are you sure you compiled for debug mode?

Oh no I didn't compile the game at all
I meant softcode or mods folder

**Reply by lemz1** [reply]
Then i guess traces won't work.

**Reply by MAZ12211** [reply]
Oh well...

**Reply by NotHyper-474** [reply]
This makes me wish they just used ``--no-traces`` instead of straight out muting `trace`

**Reply by MAZ12211** [reply]
Yup
That's more accurate

**Reply by Keoiki** [reply, marked as answer]
You need to use a debug build, it doesn't look like traces (even just for hscript) will ever be enabled again.

**Reply by MAZ12211** [reply]
Oh well...
